src/components/Pagination/Pagination.jsx

```diff
@@ -0,0 +1,28 @@
+import PropTypes from "prop-types";
+import { loopThroughPageNumbers } from "../../utils/loopThroughPageNumbers";
```

Comment:
To be honest, this function should be called `createPages` or `createPageNumbers`